pkg/controller/certificatemanager/certificatemanager.go

```diff
@@ -86,6 +87,11 @@ type CertificateManager interface {
 // Create creates a signer of new certificates and has methods to retrieve existing KeyPairs and Certificates. If a user
 // brings their own secrets, CertificateManager will preserve and return them.
 func Create(cli client.Client, installation *operatorv1.InstallationSpec, clusterDomain, ns string) (CertificateManager, error) {
+	return CreateWithLogger(cli, installation, clusterDomain, ns, log)
```

Comment:
This is a small change to allow controllers to pass their own logger when creating a certificate manager - it makes it easier in the logs to see which controller is triggering various certificate manager calls and was useful as part of my development on this PR.

I'm considering switching all controllers over to this as out-of-scope for now, though.